templates/document_head.hbs

```diff
@@ -5,8 +5,8 @@
 <script type="importmap">
 {
   "imports": {
-    "new-request-form": "{{asset 'new-request-form.js'}}",
-    "vendor": "{{asset 'vendor.js'}}"
+    "new-request-form-bundle": "{{asset 'new-request-form-bundle.js'}}",
```

Comment:
I don't like that much that we will need to always import from `xyz-bundle`, it would be nicer if the rollup plugin generates the import map as
```json
{
    "new-request-form": "{{asset 'new-request-form-bundle.js'}}"
}
```

but maybe it is going to make the plugin code more complex, so it is also ok to keep the `bundle` suffix

Reply:
I agree and maybe would prefer to keep the import name clean.
I'm going to leave it like this for now. Should be an easy change 👍🏼